### ISSUE TITLE
fullcopypage bug

### ISSUE THREAD

**Issue by cole-dda** [pymupdf/PyMuPDF#4043]
### Description of the bug

<img width="703" alt="screenshot_6579" src="https://github.com/user-attachments/assets/ce8ff92b-d807-4232-bd05-bc56f4045ce6">

res may be bytes type ，so no "m_internal" attribute

### How to reproduce the bug

[x.pdf](https://github.com/user-attachments/files/17710815/x.pdf)

import fitz
doc = fitz.open('x.pdf')
doc.fullcopy_page(0)

### PyMuPDF version

1.24.13

### Operating system

MacOS

### Python version

3.12

**Comment by JorjMcKie** [comment]
Cannot confirm: Works for me on Linux and Windows, PyMuPDF 1.24.13 and 1.24.11.

**Comment by JorjMcKie** [comment]
You did not even mention what error happened by the way.

**Comment by cole-dda** [comment]
I'am sorry
```python
    import fitz
    doc = fitz.open('x.pdf')
    doc.fullcopy_page(1)
```

**Comment by julian-smith-artifex-com** [comment]
Fixed in PyMuPDF-1.24.14.
